teuthology/nuke/actions.py

```diff
@@ -267,14 +268,26 @@ def remove_ceph_packages(ctx):
                 check_status=False,
             )
             remote.run(
-                args=['sudo', 'rpm', '--rebuilddb', run.Raw('&&'), 'yum',
-                      'clean', 'all']
-            )
-            log.info('Remove any ceph packages')
-            remote.run(
-                args=['sudo', 'yum', 'remove', '-y', run.Raw(pkgs)],
-                check_status=False
+                args=['sudo', 'rpm', '--rebuilddb']
             )
+            if dist_release in ['opensuse', 'sle']:
+                remote.run(
+                    args=['sudo', 'zypper', 'clean']
+                )
```

Comment:
any reason why `remote.run` instead of `remote.sh` ?

Reply:
yeah... looks like copy-paster

Reply:
should I replace it?

Reply:
This is a new code, why not to make it humanized?

Reply:
Could you please use _remote.sh_ like this:
```
remote.sh('sudo zippier clean')
```

Reply:
done